zone/client.cpp

```diff
@@ -5364,14 +5364,13 @@ void Client::SetStartZone(uint32 zoneid, float x, float y, float z, float headin
 	}
 
 	if (x == 0 && y == 0 && z == 0) {
-		content_db.GetSafePoints(
-			ZoneName(m_pp.binds[4].zone_id),
-			0,
-			&m_pp.binds[4].x,
-			&m_pp.binds[4].y,
-			&m_pp.binds[4].z,
-			&m_pp.binds[4].heading
-		);
+		auto zd = GetZoneVersionWithFallback(m_pp.binds[4].zone_id);
```

Comment:
GetZone() instead maybe since it's not using version here?

Reply:
Good eye, missed one